---
layout: default
title: Home
subtitle: Home
---

<div class="container">
    <!--Header/logo container-->
    <div class="row">
        <div class="col-md-8">
            <div class="content-heading clearfix media">
                <img class="pull-left" src="{{site.baseurl}}/img/plantcv_hexbadge_small.png">
                <h1>PlantCV<br><small>Plant phenotyping using computer vision</small></h1>
            </div>
        </div>
    </div>
    <!--Spacer-->
    <div class="col-md-12">
        <hr class="featurette-divider">
    </div>
    <div class="row">
        <div class="col-md-8">
            <div class="row">
                <p>PlantCV is an open-source image analysis software package targeted for plant phenotyping. The PlantCV project
                    is managed by <a href="https://www.gehan-lab.org/">Malia Gehan</a> and
                    <a href="https://www.danforthcenter.org/scientists-research/principal-investigators/noah-fahlgren">
                        Noah Fahlgren</a> and the effort of many generous contributors, collaborators, and users.
                </p>
                <h1 id="vision">Vision</h1>
                <p>Phenotyping tools will be modular and reusable such that they can be combined and recombined
                   with ease to build flexible workflows to quickly extract biologically-relevant data from images and
                   sensors. These tools will be usable by both bioinformaticians/data scientists and biologists.
                </p>
                <h1 id="mission">Mission</h1>
                <p>
                    <ul>
                        <li>Provide a common interface for a collection of image analysis techniques that are
                            integrated from a variety of source packages and algorithms.
                        </li>
                        <li>Utilize a modular architecture that enables flexibility in the design of analysis workflows
                            and rapid assimilation and integration of new methods.
                        </li>
                        <li>Develop a network of users, collaborators, and contributors by developing openly in
                            real-time in the cloud using an open-source framework to rapidly disseminate new methods.
                        </li>
                        <li>Provide a simplified interface for users to utilize the underlying tools and build custom
                            analysis workflows without significant experience with programming.
                        </li>
                        <li>Utilize PlantCV as a tool for training researchers and students in image and data analysis
                            techniques.
                        </li>
                    </ul>
                </p>
                <h1 id="values">Values</h1>
                <p>The PlantCV project values open communication and collaboration among stakeholders from diverse
                    backgrounds and areas of expertise. Through the PlantCV project we seek to highlight the valuable
                    ideas and contributions of members of the community.
                </p>
            </div>
            <div class="row">
                <h1 id="about">About</h1>
                <p>PlantCV is an open-source image analysis software package targeted for plant phenotyping. The PlantCV project
                    was started at the Donald Danforth Plant Science Center in 2014, and is under active
                    development—new functionality and tutorials are added regularly. Keep up to date on our progress
                    via twitter:
                    <a href="https://twitter.com/plantcv" class="twitter-follow-button" data-show-count="false">
                        Follow @plantcv
                    </a>, or follow us on <a href="https://github.com/danforthcenter/plantcv">GitHub</a>.
                </p>
                <p>If you have questions, bug reports, suggestions for new features, etc. please post to our
                    <a href="https://github.com/danforthcenter/plantcv/issues">GitHub issues page</a>.
                </p>
                <p>We encourage community contribution. See our
                    <a href="https://plantcv.readthedocs.io/en/latest/CONTRIBUTING/">guide for contributors</a>, our
                    <a href="https://plantcv.readthedocs.io/en/latest/CODE_OF_CONDUCT/">code of conduct</a>, and
                    current <a href="https://github.com/danforthcenter/plantcv/graphs/contributors">list of contributors
                    </a>.
                </p>
                <h1 id="presentations">Presentations/Workshops</h1>
                <iframe width="560" height="315" src="https://www.youtube.com/embed/zvn_05cE0L4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                <iframe width="560" height="315" src="https://www.youtube.com/embed/fVoPjvgT400" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                <iframe src="https://player.vimeo.com/video/264180642" width="640" height="360" frameborder="0" allowfullscreen></iframe>
                <iframe width="560" height="315" src="https://www.youtube.com/embed/zEL3YRPIqnc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                <iframe width="560" height="315" src="https://www.youtube.com/embed/LGvZxy4avOE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
                <iframe allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="" frameborder="0" height="315" src="https://www.youtube.com/embed/WCinnxUnqRo" width="560"></iframe>
                <h1 id="plantcv-publications">PlantCV Publications</h1>
                <p>If you use PlantCV, please cite the appropriate PlantCV publication below.</p>
                <ol>
                    <li>
                        Casto A, Schuhl H, Schneider D, Wheeler J, Gehan M, Fahlgren N. 2021. Analyzing chlorophyll
                        fluorescence images in PlantCV. <i>Earth and Space Science Open Archive</i>. DOI:
                        <a href="https://doi.org/10.1002/essoar.10508322.2">10.1002/essoar.10508322.2</a>.
                    </li>
                    <li>
                        Gutierrez Ortega JA, Castillo SE, Gehan M, Fahlgren N. 2021. Segmentation of overlapping plants
                        in multi-plant image time series. <i>Earth and Space Science Open Archive</i>. DOI:
                        <a href="https://doi.org/10.1002/essoar.10508337.2">10.1002/essoar.10508337.2</a>.
                    </li>
                    <li>
                        Hodge JG, Li Q, Doust A. 2021. De novo homology assessment from landmark data: A workflow to
                        identify and track segmented structures in plant time series images.
                        <i>bioRxiv</i>:2021.02.21.432162. DOI:
                        <a href="https://doi.org/10.1101/2021.02.21.432162">10.1101/2021.02.21.432162</a>.
                    </li>
                    <li>Berry JC, Fahlgren N, Pokorny AA, Bart RS, Veley KM. 2018. An automated, high-throughput
                        method for standardizing image color profiles to improve image-based plant phenotyping.
                        <i>PeerJ</i> 6:e5727. DOI: <a href="https://doi.org/10.7717/peerj.5727">10.7717/peerj.5727</a>.
                    </li>
                    <li>Gehan MA*, Fahlgren N*, Abbasi A, Berry JC, Callen ST, Chavez L, Doust AN, Feldman MJ, Gilbert
                        KB, Hodge JG, Hoyer JS, Lin A, Liu S, Lizárraga C, Lorence A, Miller M, Platon E, Tessman M,
                        Sax T. 2017. PlantCV v2: Image analysis software for high-throughput plant phenotyping.
                        <i>PeerJ</i> 5:e4088. DOI: <a href="https://doi.org/10.7717/peerj.4088">10.7717/peerj.4088</a>.
                    </li>
                    <li>Abbasi A, Fahlgren N. 2016. Naive Bayes pixel-level plant segmentation. In: <i>2016 IEEE Western
                        New York Image and Signal Processing Workshop (WNYISPW)</i>. 1–4. DOI:
                        <a href="https://doi.org/10.1109/WNYIPW.2016.7904790">10.1109/WNYIPW.2016.7904790</a>.
                    </li>
                    <li>Fahlgren N*, Feldman M*, Gehan MA*, Wilson MS, Shyu C, Bryant DW, Hill ST, McEntee CJ,
                        Warnasooriya SN, Kumar I, Ficor T, Turnipseed S, Gilbert KB, Brutnell TP, Carrington JC,
                        Mockler TC, Baxter I. 2015. A versatile phenotyping system and analytics platform reveals
                        diverse temporal responses to water availability in <i>Setaria</i>. <i>Molecular Plant</i>
                        8:1520–1535. DOI:
                        <a href="https://doi.org/10.1016/j.molp.2015.06.005">10.1016/j.molp.2015.06.005.</a>
                    </li>
                </ol>
                <h1 id="publications-using-plantcv">Publications Using PlantCV</h1>
                <ol>
                    <li>
                        Wang W, Talide L, Viljamaa S, Niittylä T. 2022. Aspen growth is not limited by starch reserves.
                        <i>Current Biology</i>: CB.
                        DOI: <a href="https://doi.org/10.1016/j.cub.2022.06.056">10.1016/j.cub.2022.06.056</a>.
                    </li>
                    <li>
                        Beyene G, Chauhan RD, Villmer J, Husic N, Wang N, Gebre E, Girma D, Chanyalew S, Assefa K, Tabor G,
                        Gehan M, McGrone M, Yang M, Lenderts B, Schwartz C, Gao H, Gordon-Kamm W, Taylor NJ, MacKenzie DJ.
                        2022. CRISPR/Cas9-mediated tetra-allelic mutation of the “Green Revolution” SEMIDWARF-1 (SD-1) gene
                        confers lodging resistance in Tef (Eragrostis tef). <i>Plant Biotechnology Journal</i>.
                        DOI: <a href="https://doi.org/10.1111/pbi.13842">10.1111/pbi.13842</a>.
                    </li>
                    <li>
                        Kinose R, Utsumi Y, Iwamura M, Kise K. 2022. Tiller estimation method using deep neural networks.
                        DOI: <a href="https://doi.org/10.21203/rs.3.rs-1552723/v1">10.21203/rs.3.rs-1552723/v1</a>.
                    </li>
                    <li>
                        Castillo SE, Tovar JC, Shamin A, Gutirerrez J, Pearson P, Gehan MA. 2022. A protocol for <i>Chenopodium
                        quinoa</i> pollen germination. <i>Plant Methods</i> 18:65. DOI:
                        <a href="https://doi.org/10.1186/s13007-022-00900-3">10.1186/s13007-022-00900-3</a>.
                    </li>
                    <li>
                        Marrano A, Moyers BT. 2022. Scanning the rice Global MAGIC population for dynamic genetic control of
                        seed traits under vegetative drought. <i>The Plant Phenome Journal</i> 5. DOI:
                        <a href="https://doi.org/10.1002/ppj2.20033">10.1002/ppj2.20033</a>.
                    </li>
                    <li>
                        Tanaka K, Kato Y, Mikawa M, Fujisawa M. 2022. Dynamic grass color scale display technique based on
                        grass length for green landscape-friendly animation display. <i>arXiv</i>:2203.08496 [cs.GR].
                        <a href="http://arxiv-export3.library.cornell.edu/abs/2203.08496">http://arxiv-export3.library.cornell.edu/abs/2203.08496</a>.
                    </li>
                    <li>
                        Arunachalam A, Andreasson H. 2022. MSI-RPi: Affordable, portable, and modular multispectral imaging
                        prototype suited to operate in UV, visible and mid-infrared regions. <i>Journal of Mobile
                        Multimedia</i>:723–742. DOI:
                        <a href="https://doi.org/10.13052/jmm1550-4646.18312">10.13052/jmm1550-4646.18312</a>.
                    </li>
                    <li>
                        Scandola S, Mehta D, Li Q, Rodriguez Gallo MC, Castillo B, Uhrig RG. 2022. Multi-omic analysis shows
                        REVEILLE clock genes are involved in carbohydrate metabolism and proteasome function.
                        <i>Plant Physiology</i>. DOI: <a href="https://doi.org/10.1093/plphys/kiac269">10.1093/plphys/kiac269</a>.
                    </li>
                    <li>
                        Chang L, Li D, Hameed MK, Yin Y, Huang D, Niu Q. 2021. Using a hybrid neural network model DCNN–LSTM
                        for image-based nitrogen nutrition diagnosis in muskmelon. <i>Horticulturae</i> 7:489.
                        DOI: <a href="https://doi.org/10.3390/horticulturae7110489">10.3390/horticulturae7110489</a>.
                    </li>
                    <li>
                        Afzali S, Mosharafian S, van Iersel MW, Mohammadpour Velni J. 2021. Development and implementation of
                        an IoT-enabled optimal and predictive lighting control strategy in greenhouses. <i>Plants</i> 10:2652.
                        DOI: <a href="https://doi.org/10.3390/plants10122652">10.3390/plants10122652</a>.
                    </li>
                    <li>
                        Polydore S, Fahlgren N. 2021. Phenotypic analysis of a European <i>Camelina sativa</i>
                        diversity panel. <i>Earth and Space Science Open Archive</i>. DOI:
                        <a href="https://doi.org/10.1002/essoar.10508336.2">10.1002/essoar.10508336.2</a>.
                    </li>
                    <li>
                        Teng C, Fahlgren N, Meyers BC. 2021. Tasselyzer, a machine learning method to quantify
                        anther extrusion in maize, based on PlantCV. <i>bioRxiv</i>:2021.09.27.461799.
                        DOI: <a href="https://doi.org/10.1101/2021.09.27.461799">10.1101/2021.09.27.461799</a>.
                    </li>
                    <li>
                        Roquis D, Robertson M, Yu L, Thieme M, Julkowska M, Bucher E. 2021. Genomic impact of
                        stress-induced transposable element mobility in Arabidopsis. <i>Nucleic Acids Research</i>.
                        DOI: <a href="https://doi.org/10.1093/nar/gkab828">10.1093/nar/gkab828</a>.
                    </li>
                    <li>
                        Cox KL, Manchego J, Meyers BC, Czymmek KJ, Harkess A. 2021. Automated imaging of duckweed
                        growth and development. <i>bioRxiv</i>:2021.07.21.453240. DOI:
                        <a href="https://doi.org/10.1101/2021.07.21.453240">10.1101/2021.07.21.453240</a>.
                    </li>
                    <li>
                        Huber M, Julkowska MM, Snoek BL, van Veen H, Toulotte J, Kumar V, Kajala K, Sasidharan R, Pierik R.
                        2021. Towards increased shading potential: a combined phenotypic and genetic analysis of rice shoot
                        architecture. bioRxiv:2021.05.25.445664. DOI:
                        <a href="https://doi.org/10.1101/2021.05.25.445664">10.1101/2021.05.25.445664</a>.
                    </li>
                    <li>
                        Renaud JB, DesRochers N, Hoogstra S, Garnham CP, Sumarah MW. 2021. Structure activity
                        relationship for fumonisin phytotoxicity. <i>Chemical Research in Toxicology</i> 34:1604–1611.
                        DOI: <a href="https://doi.org/10.1021/acs.chemrestox.1c00057">10.1021/acs.chemrestox.1c00057</a>.
                    </li>
                    <li>
                        Li Q, Liu N, Liu Q, Zheng X, Lu L, Gao W, Liu Y, Liu Y, Zhang S, Wang Q, Pan J, Chen C, Mi Y,
                        Yang M, Cheng X, Ren G, Yuan Y-W, Zhang X. 2021. DEAD-box helicases modulate dicing body
                        formation in Arabidopsis. <i>Science Advances</i> 7. DOI:
                        <a href="https://doi.org/10.1126/sciadv.abc6266">10.1126/sciadv.abc6266</a>.
                    </li>
                    <li>
                        van de Koot WQM, van Vliet LJJ, Chen W, Doonan JH, Nibau C. 2021. Development of an image
                        analysis pipeline to estimate sphagnum colony density in the field. Plants 10. DOI:
                        <a href="https://doi.org/10.3390/plants10050840">10.3390/plants10050840</a>.
                    </li>
                    <li>
                        Badhan S, Desai K, Dsilva M, Sonkusare R, Weakey S. 2021. Real-time weed detection using
                        machine learning and stereo-vision. In: 2021 6th International Conference for Convergence
                        in Technology (I2CT). 1–5. DOI: <a href="https://doi.org/10.1109/I2CT51068.2021.9417989">10.1109/I2CT51068.2021.9417989</a>.
                    </li>
                    <li>
                        Palermo F, Oh C, Althoefer K, Poslad S, Farkhatdinov I. 2021. Investigation of images of cracks
                        via graph theory for developing an optimal exploration algorithm for a robotic manipulator. In:
                        2021 International Conference on Robotics and Automation. IEEE.
                    </li>
                    <li>
                        Kienbaum L, Abondano MC, Blas R, Schmid K. 2021. DeepCob: Precise and high-throughput analysis
                        of maize cob geometry using deep learning with an application in genebank phenomics.
                        <i>bioRxiv</i>:2021.03.16.435660. DOI:
                        <a href="https://doi.org/10.1101/2021.03.16.435660">10.1101/2021.03.16.435660</a>.
                    </li>
                    <li>
                        Al-Lami MK, Nguyen D, Oustriere N, Burken JG. 2021. High throughput screening of native species
                        for tailings eco-restoration using novel computer visualization for plant phenotyping. <i>The
                        Science of the Total Environment</i> 780:146490. DOI:
                        <a href="https://doi.org/10.1016/j.scitotenv.2021.146490">10.1016/j.scitotenv.2021.146490</a>.
                    </li>
                    <li>
                        Kim J, Go S, Noh K, Park S, Lee S. 2021. Fully leveraging deep learning methods for
                        constructing retinal fundus photomontages. <i>Applied Sciences</i> 11:1754. DOI:
                        <a href="https://doi.org/10.3390/app11041754">10.3390/app11041754</a>.
                    </li>
                    <li>
                        Zhang X, Wang D, Elberse J, Qi L, Shi W, Peng Y-L, Schuurink RC, Van den Ackerveken G, Liu J.
                        2021. Structure-guided analysis of the Arabidopsis JASMONATE-INDUCED OXYGENASE (JOX) 2 reveals
                        key residues of plant JOX recognizing jasmonic acid substrate. <i>Molecular Plant</i>. DOI:
                        <a href="https://doi.org/10.1016/j.molp.2021.01.017">10.1016/j.molp.2021.01.017</a>.
                    </li>
                    <li>
                        Fernández Nevyl S, Battaglia ME. 2021. Developmental plasticity in <i>Arabidopsis thaliana</i>
                        under combined cold and water deficit stresses during flowering stage. <i>Planta</i> 253:50.
                        DOI: <a href="https://doi.org/10.1007/s00425-021-03575-7">10.1007/s00425-021-03575-7</a>.
                    </li>
                    <li>
                        Paradis OP, Jessurun NT, Tehranipoor M, Asadizanjani N. 2020. Color normalization for robust
                        automatic bill of materials generation and visual inspection of PCBs. In: <i>ISTFA 2020: Papers
                        Accepted for the Planned 46th International Symposium for Testing and Failure Analysis</i>.
                        ASM International,. DOI:
                        <a href="https://doi.org/10.31399/asm.cp.istfa2020p0172">10.31399/asm.cp.istfa2020p0172</a>.
                    </li>
                    <li>
                        Nurminen A, Malhi A. 2020. Green thumb engineering: Artificial intelligence for managing IoT
                        enabled houseplants. In: <i>2020 IEEE Global Conference on Artificial Intelligence and Internet
                        of Things (GCAIoT)</i>. 01–07. DOI:
                        <a href="https://doi.org/10.1109/GCAIoT51063.2020.9345850">10.1109/GCAIoT51063.2020.9345850</a>.
                    </li>
                    <li>
                        White AE, Dikow RB, Baugh M, Jenkins A, Frandsen PB. 2020. Generating segmentation masks of
                        herbarium specimens and a data set for training segmentation models using deep learning.
                        <i>Applications in Plant Sciences</i> 8:e11352. DOI:
                        <a href="https://doi.org/10.1002/aps3.11352">10.1002/aps3.11352</a>.
                    </li>
                    <li>
                        Kumar D, Kushwaha S, Delvento C, Liatukas Ž, Vivekanand V, Svensson JT, Henriksson T,
                        Brazauskas G, Chawade A. 2020. Affordable phenotyping of winter wheat under field and
                        controlled conditions for drought tolerance. <i>Agronomy</i> 10:882. DOI:
                        <a href="https://doi.org/10.3390/agronomy10060882">10.3390/agronomy10060882</a>.
                    </li>
                    <li>
                        Teng C, Zhang H, Hammond R, Huang K, Meyers BC, Walbot V. 2020. Dicer-like 5 deficiency confers
                        temperature-sensitive male sterility in maize. <i>Nature Communications</i> 11:2912. DOI:
                        <a href="https://doi.org/10.1038/s41467-020-16634-6">10.1038/s41467-020-16634-6</a>.
                    </li>
                    <li>
                        Acosta-Gamboa LM, Suxing L, Jarrod W C, Zachary C C, Raquel T, Walter P S, Jessica P Y-C,
                        Lorence A. 2020. Characterization of the response to abiotic stresses of high ascorbate
                        Arabidopsis lines using phenomic approaches. <i>Plant Physiology and Biochemistry</i>
                        151:500–515. DOI:
                        <a href="https://doi.org/10.1016/j.plaphy.2020.03.038">10.1016/j.plaphy.2020.03.038</a>.
                    </li>
                    <li>
                        Tovar JC, Quillatupa C, Callen ST, Castillo SE, Pearson P, Shamin A, Schuhl H, Fahlgren N,
                        Gehan MA. 2020. Heating quinoa shoots results in yield loss by inhibiting fruit production and
                        delaying maturity. <i>The Plant Journal: For Cell and Molecular Biology</i> 102:1058–1073. DOI:
                        <a href="https://doi.org/10.1111/tpj.14699">10.1111/tpj.14699</a>.
                    </li>
                    <li>
                        Schneider D, Lopez LS, Li M, Crawford JD, Kirchhoff H, Kunz H-H. 2019. Fluctuating light
                        experiments and semi-automated plant phenotyping enabled by self-built growth racks and
                        simple upgrades to the IMAGING-PAM. <i>Plant Methods</i> 15:156. DOI:
                        <a href="https://doi.org/10.1186/s13007-019-0546-1">10.1186/s13007-019-0546-1</a>.
                    </li>
                    <li>
                        Shakoor N, Agnew E, Ziegler G, Lee S, Lizarraga C, Fahlgren N, Baxter I, Mockler TC. 2019.
                        Genomewide association study reveals transient loci underlying the genetic architecture of
                        biomass accumulation under cold stress in Sorghum. <i>bioRxiv</i>:760025. DOI:
                        <a href="https://doi.org/10.1101/760025">10.1101/760025</a>.
                    </li>
                    <li>
                        Zheng X, Fahlgren N, Abbasi A, Berry JC, Carrington JC. 2019. Antiviral ARGONAUTEs against
                        <i>Turnip Crinkle Virus</i> revealed by image-based trait analysis. <i>Plant Physiology</i>
                        180:1418–1435. DOI: <a href="https://doi.org/10.1104/pp.19.00121">10.1104/pp.19.00121</a>.
                    </li>
                    <li>Enders TA, St. Dennis S, Oakland J, Callen ST, Gehan MA, Miller ND, Spalding EP, Springer NM,
                        Hirsch CD. 2019. Classifying cold-stress responses of inbred maize seedlings using RGB imaging.
                        <i>Plant Direct</i> 3:e00104. DOI:
                        <a href="https://doi.org/10.1002/pld3.104">10.1002/pld3.104</a>.
                    </li>
                    <li>Feldman MJ, Ellsworth PZ, Fahlgren N, Gehan MA, Cousins AB, Baxter I. 2018. Components of
                        water use efficiency have unique genetic signatures in the model C4 grass <i>Setaria</i>.
                        <i>Plant Physiology</i> 178:699–715. DOI:
                        <a href="https://doi.org/10.1104/pp.18.00146">10.1104/pp.18.00146</a>.
                    </li>
                    <li>Armoniené R, Odilbekov F, Vivekanand V, Chawade A. 2018. Affordable imaging lab for noninvasive
                        analysis of biomass and early vigour in cereal crops. <i>BioMed Research International</i> 2018.
                        DOI: <a href="https://doi.org/10.1155/2018/5713158">10.1155/2018/5713158</a>.
                    </li>
                    <li>Tovar JC*, Hoyer JS*, Lin A, Tielking A, Callen ST, Castillo SE, Miller M, Tessman M, Fahlgren
                        N, Carrington JC, Nusinow DA, Gehan MA. 2018. Raspberry Pi-powered imaging for plant
                        phenotyping. <i>Applications in Plant Sciences</i> 6:e1031. DOI:
                        <a href="https://doi.org/10.1002/aps3.1031">10.1002/aps3.1031</a>.
                    </li>
                    <li>Liang Z, Pandey P, Stoerger V, Xu Y, Qiu Y, Ge Y, Schnable JC. 2018. Conventional and
                        hyperspectral time-series imaging of maize lines widely used in field trials. <i>GigaScience</i>
                        7:1–11. DOI:
                        <a href="https://doi.org/10.1093/gigascience/gix117">10.1093/gigascience/gix117</a>.
                    </li>
                    <li>Veley KM, Berry JC, Fentress SJ, Schachtman DP, Baxter I, Bart R. 2017. High-throughput
                        profiling and analysis of plant responses over time to abiotic stress. <i>Plant Direct</i>
                        1:e00023. DOI: <a href="https://doi.org/10.1002/pld3.23">10.1002/pld3.23</a>.
                    </li>
                    <li>Liu S, Acosta-Gamboa LM, Huang X, Lorence A. 2017. Novel low cost 3D surface model
                        reconstruction system for plant phenotyping. <i>Journal of Imaging</i> 3:39. DOI:
                        <a href="https://doi.org/10.3390/jimaging3030039">10.3390/jimaging3030039</a>.
                    </li>
                    <li>Ubbens JR, Stavness I. 2017. Deep Plant Phenomics: A deep learning platform for complex plant
                        phenotyping tasks. Frontiers in Plant Science 8:1190. DOI:
                        <a href="https://doi.org/10.3389/fpls.2017.01190">10.3389/fpls.2017.01190</a>.
                    </li>
                    <li>Feldman MJ, Paul RE, Banan D, Barrett JF, Sebastian J, Yee M-C, Jiang H, Lipka AE, Brutnell TP,
                        Dinneny JR, Leakey ADB, Baxter I. 2017. Time dependent genetic analysis links field and
                        controlled environment phenotypes in the model C4 grass <i>Setaria</i>. <i>PLoS Genetics</i>
                        13:e1006841. DOI:
                        <a href="https://doi.org/10.1371/journal.pgen.1006841">10.1371/journal.pgen.1006841</a>.
                    </li>
                    <li>Gehan MA, Kellogg EA. 2017. High-throughput phenotyping. <i>American Journal of Botany</i>
                        104:505–508. DOI: <a href="https://doi.org/10.3732/ajb.1700044">10.3732/ajb.1700044</a>.
                    </li>
                    <li>Agnew E*, Bray A*, Floro E*, Ellis N, Gierer J, Lizárraga C, O’Brien D, Wiechert M, Mockler TC,
                        Shakoor N, Topp CN. 2016. Whole-plant manual and image-based phenotyping in controlled
                        environments. In: <i>Current Protocols in Plant Biology</i>. John Wiley & Sons, Inc. DOI:
                        <a href="https://doi.org/10.1002/cppb.20044">10.1002/cppb.20044</a>.
                    </li>
                </ol>
            </div>
        </div>
        <div class="col-md-4">
            <h2 id=stats>Statistics</h2>
            <a href="https://github.com/danforthcenter/plantcv">
                <img alt="GitHub contributors" src="https://img.shields.io/github/contributors/danforthcenter/plantcv?label=GitHub contributors">
            </a><br>
            <a href="https://plantcv.danforthcenter.org/#plantcv-publications">
                <img alt="Publications" src="https://img.shields.io/endpoint?url=https%3A%2F%2Fplantcv.danforthcenter.org%2Fbadges%2Fpapers.json">
            </a><br>
            <a href="https://anaconda.org/conda-forge/plantcv">
                <img alt="Conda" src="https://img.shields.io/conda/dn/conda-forge/plantcv?label=conda%20downloads">
            </a><br>
            <a href="https://pypi.org/project/plantcv">
                <img alt="PyPI - Downloads" src="https://img.shields.io/pypi/dd/plantcv?label=PyPI%20downloads">
            </a><br>
            <a href="https://github.com/danforthcenter/plantcv">
                <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/danforthcenter/plantcv?style=social">
            </a>
            <a href="https://twitter.com/plantcv">
                <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/plantcv?style=social">
            </a>
            <hr class="featurette-divider">
            <a class="twitter-timeline" data-height="2500" href="https://twitter.com/plantcv?ref_src=twsrc%5Etfw">Tweets by plantcv</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
        </div>
    </div>
    <!--Spacer-->
    <div class="col-md-12">
        <hr class="featurette-divider">
    </div>
